### ISSUE TITLE
Too many redirects error when logging into multiple apps

### ISSUE THREAD

**Issue by james-martin-jd** [solid/node-solid-server#1352]
Ran into an error trying to log in, specifically to the inrupt.net provider. 

To reproduce, I logged first into develop.generator.inrupt.com using my jmartin.inrupt.net pod, then logged into localhost as the same pod. The first login was successful (and showed the login form on the provider as expected). The second login immediately showed a Chrome error, reading:
```
This page isn’t working
inrupt.net redirected you too many times.
Try clearing your cookies.
ERR_TOO_MANY_REDIRECTS
```
Clearing my cookies did indeed help, but I was able to reproduce this error a second time.

**Comment by jaxoncreed** [comment]
Right, this is probably some unintended consequence of the `user not logged in 2` bug fix.
